frontend/dispatcher.lua

```diff
@@ -12,37 +13,241 @@ local Dispatcher = {
+    toc = { category="none", event="ShowToc", title=_("Table of contents"), rolling=true, paging=true,},
+    bookmarks = { category="none", event="ShowBookmark", title=_("Bookmarks"), rolling=true, paging=true,},
+    book_statistics = { category="none", event="ShowBookStats", title=_("Book statistics"), rolling=true, paging=true,},
+    book_status = { category="none", event="ShowBookStatus", title=_("Book status"), rolling=true, paging=true,},
+    book_info = { category="none", event="ShowBookInfo", title=_("Book information"), rolling=true, paging=true,},
+    book_description = { category="none", event="ShowBookDescription", title=_("Book description"), rolling=true, paging=true,},
+    book_cover = { category="none", event="ShowBookCover", title=_("Book cover"), rolling=true, paging=true,},
+    show_config_menu = { category="none", event="ShowConfigMenu", title=_("Show bottom menu"), rolling=true, paging=true,},
+    toggle_bookmark = { category="none", event="ToggleBookmark", title=_("Toggle bookmark"), rolling=true, paging=true,},
+    toggle_inverse_reading_order = { category="none", event="ToggleReadingOrder", title=_("Toggle page turn direction"), rolling=true, paging=true,},
+    cycle_highlight_action = { category="none", event="CycleHighlightAction", title=_("Cycle highlight action"), rolling=true, paging=true,},
+    cycle_highlight_style = { category="none", event="CycleHighlightStyle", title=_("Cycle highlight style"), rolling=true, paging=true,},
+    kosync_push_progress = { category="none", event="KOSyncPushProgress", title=_("Push progress from this device"), rolling=true, paging=true,},
+    kosync_pull_progress = { category="none", event="KOSyncPullProgress", title=_("Pull progress from other devices"), rolling=true, paging=true,},
+    page_jmp = { category="absolutenumber", event="GotoViewRel" , min=-100, max=100, title=_("Go X pages"), rolling=true, paging=true,},
+
+    --rolling reader settings
+    increase_font = { category="incrementalnumber", event="IncreaseFontSize", min=1, max=255, title=_("Increase font size"), rolling=true,},
+    decrease_font = { category="incrementalnumber", event="DecreaseFontSize", min=1, max=255, title=_("Decrease font size"), rolling=true,},
+
+    --paging reader settings
+    toggle_page_flipping = { category="none", event="TogglePageFlipping", title=_("Toggle page flipping"), paging=true,},
+    toggle_reflow = { category="none", event="ToggleReflow", title=_("Toggle reflow"), paging=true,},
+    zoom = { category="string", event="SetZoomMode", title=_("Zoom to"), args={"contentwidth", "contentheight", "pagewidth", "pageheight", "column", "content", "page"}, toggle={"content width", "content height", "page width", "page height", "column", "content", "page"}, paging=true,},
+
+    --parsed from CreOptions
+    rotation_mode = {category="string", device=true},
+    visible_pages = {category="string", rolling=true},
+    h_page_margins = {category="string", rolling=true},
```

Comment:
No `title=` for all of these ?
If it is pulled from elsewhere, might be worth a comment, so I don't have to look at the rest of the code (which I just quickly did and did not found anything).

Reply:
as per the comment `-- parsed from CreOptions`